include/boost/http_proto/impl/parser.hpp

```diff
@@ -7,10 +7,14 @@
 // Official repository: https://github.com/cppalliance/http_proto
 //
 
+#pragma once
```

Comment:
you might put a comment here explaining the circular dependency messing up VS Code